vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/card/conection/InputCardNumberConnection.kt

```diff
@@ -12,7 +12,7 @@ import com.verygoodsecurity.vgscollect.view.card.validation.LengthMatchValidator
 internal class InputCardNumberConnection(
     id: Int,
     validator: CompositeValidator,
-    private val IcardBrand: IDrawCardBrand? = null,
+    private val iCardBrand: IDrawCardBrand? = null,
```

Comment:
@DonaldRG Can we remove `i` and name this val just `cardBrand`?

Reply:
That make sense 👍